ext/gd/gd.c

```diff
@@ -917,12 +917,12 @@ PHP_FUNCTION(imagecreatetruecolor)
 	}
 
 	if (x_size <= 0 || x_size >= INT_MAX) {
-		zend_value_error("Invalid width (x_size)");
```

Comment:
I'm not sure if showing the exact interval would be better

Reply:
Saying greater than 0 should be fine IMHO